pydocmd/loader.py

```diff
@@ -31,117 +31,122 @@
 from yapf.yapflib.yapf_api import FormatCode
 
 function_types = (types.FunctionType, types.LambdaType, types.MethodType,
-  types.BuiltinFunctionType, types.BuiltinMethodType)
+                  types.BuiltinFunctionType, types.BuiltinMethodType)
 if hasattr(types, 'UnboundMethodType'):
-  function_types += (types.UnboundMethodType,)
+    function_types += (types.UnboundMethodType,)
 
 
 def trim(docstring):
-  if not docstring:
-    return ''
-  lines = [x.rstrip() for x in docstring.split('\n')]
-  lines[0] = lines[0].lstrip()
-
-  indent = None
-  for i, line in enumerate(lines):
-    if i == 0 or not line: continue
-    new_line = line.lstrip()
-    delta = len(line) - len(new_line)
-    if indent is None:
-      indent = delta
-    elif delta > indent:
-      new_line = ' ' * (delta - indent) + new_line
-    lines[i] = new_line
-
-  return '\n'.join(lines)
+    if not docstring:
+        return ''
+    lines = [x.rstrip() for x in docstring.split('\n')]
+    lines[0] = lines[0].lstrip()
+
+    indent = None
+    for i, line in enumerate(lines):
+        if i == 0 or not line:
+            continue
+        new_line = line.lstrip()
+        delta = len(line) - len(new_line)
+        if indent is None:
+            indent = delta
+        elif delta > indent:
+            new_line = ' ' * (delta - indent) + new_line
+        lines[i] = new_line
+
+    return '\n'.join(lines)
```

Comment:
Did the line terminator change here?

Reply:
Ah no the indentation changed. Please go back to two-space, I know it's not common but the rest of the source is currently using a two-space indentation. I'd rather have this PR contain only the changes it intends to address, and not _also_ try to perform cleanup

Reply:
Will do